setup.py

```diff
@@ -11,11 +11,11 @@
 
 setup(
     name="nexmo",
-    version="2.5.0",
-    description="Nexmo Client Library for Python",
+    version="2.5.1",
+    description="Vonage Client Library for Python",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="https://github.com/Nexmo/nexmo-python",
```

Comment:
Since this needs to be pushed to the Nexmo repo, we should keep this as Nexmo branding for the moment. When we push a Vonage package we can update this.